pkg/controller/directimagemigration/task.go

```diff
@@ -207,6 +209,15 @@ func (t *Task) Run() error {
 
 // Advance the task to the next phase.
 func (t *Task) next() error {
+	// Write time taken to complete phase
+	t.Owner.Status.StageCondition(migapi.Running)
+	cond := t.Owner.Status.FindCondition(migapi.Running)
+	if cond != nil {
+		now := time.Now().UTC()
+		elapsed := now.Sub(cond.LastTransitionTime.Time.UTC())
+		t.Log.Info("[NEXT]", "Elapsed", elapsed)
```

Comment:
I wonder if instead of `[NEXT]` the `[COMPLETE]` would make sense?

Just trying to think of how the log would read to a normal user no aware of why this is for? [COMPLETE] with a phase name and time will give them a good idea of why this log entry exists, I guess.

Reply:
Since `[COMPLETED]` is already used when the entire migration completes I felt it was too similar, although that was my first intuition as well. 

For some context, the task logger has the a KV `"Phase": "<PhaseName>"` attached. 

I was considering some alternatives like `[PHASECOMPLETE]` but that seemed maybe a bit long? WDYT

Reply:
I guess I should note we could just do a message like this
```
{"level":"info","ts":[...],"logger":"[...]","msg":"Phase completed.",[...],"Phase": "StartRefresh", "Elapsed":0.522302}
```

Reply:
Changed to 
```
{"level":"info","ts":[...],"logger":"[...]","msg":"Phase completed",[...],"Phase": "StartRefresh", "Elapsed":0.522302}

Reply:
the latest change works with me, thanks @djwhatle